modules/http-load-balancer/variables.tf

```
@@ -34,6 +34,12 @@ variable "ssl_certificates" {
   default     = []
 }
 
+variable "ssl_policy" {
+  description = "Link to ssl policy. Omitted if not provided"
+  type        = string
+  default     = ""
```

```suggestion
  default     = null
```
Let's set the default to `null` for better consistency with our other modules.